docs/docsite/rst/dev_guide/developing_collections_testing.rst

```diff
@@ -19,7 +19,15 @@ Use ``ansible-test`` tool from your collection directory, which must include ``a
 
 You can run several sanity tests, as well as run unit and integration tests for plugins using ``ansible-test``. When you test collections, test against the ansible-core version(s) you are targeting.
 
-You must always execute ``ansible-test`` from the root directory of a collection. You can run ``ansible-test`` in Docker containers without installing any special requirements. The Ansible team uses this approach in Azure Pipelines both in the ansible/ansible GitHub repository and in the large community collections such as `community.general <https://github.com/ansible-collections/community.general/>`_ and `community.network <https://github.com/ansible-collections/community.network/>`_. The examples below demonstrate running tests in Docker containers.
+You must always execute ``ansible-test`` from the root directory of a collection. You can run ``ansible-test`` in Docker containers without installing any special requirements. The Ansible team uses this approach in Azure Pipelines both in the ansible/ansible GitHub repository and in the large community collections such as `community.general <https://github.com/ansible-collections/community.general/>`_ and `community.network <https://github.com/ansible-collections/community.network/>`_ to automatically run the tests when pull requests are submitted.
+
+Many collections which do not require running tests on different OS distributions use
+GitHub Actions as their continuous integration (CI) platform.
+The `collection_template repository <https://github.com/ansible-collections/collection_template>`_ contains
+GitHub Actions workflow `templates <https://github.com/ansible-collections/collection_template/tree/main/.github/workflows>`_
+that collection developers are free to use to easily set up CI in their collection repositories.
```

Comment:
As above I would encourage keeping the entire sentence on one line.

Reply:
SGTM, please suggest the changes and commit